### ISSUE TITLE
Train on own dataset 

### ISSUE THREAD

**Issue by MustafaYasin** [JingyunLiang/VRT#44]
Hi, while I'm trying to test the model on my own dataset, but I'm still running into the same issue:

/usr/local/lib/python3.7/dist-packages/torch/functional.py:478: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  ../aten/src/ATen/native/TensorShape.cpp:2894.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
/content/VRT/models/network_vrt.py:716: UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').
  dim_t = temperature ** (2 * (dim_t // 2) / num_pos_feats)
loading model from ./model_zoo/vrt/model_zoo/vrt/001_VRT_videosr_bi_REDS_6frames.pth
using dataset from testsets/uploaded
tcmalloc: large alloc 4187750400 bytes == 0x4d22e000 @  0x7fcae3624b6b 0x7fcae3644379 0x7fca5f668d57 0x7fca5f656bc3 0x7fca8954c39f 0x7fca8954cd10 0x7fca8954cd64 0x7fca89a5dfff 0x7fca8a2ce89b 0x7fca8a01a223 0x7fca8a2a99bf 0x7fca8a057bf7 0x7fcab15aac30 0x7fca89a653b4 0x7fca8a51c515 0x7fca89dc49ce 0x7fca8a2a81a5 0x7fca89e0a372 0x7fcab128bad7 0x593784 0x548c51 0x51566f 0x593dd7 0x5118f8 0x593dd7 0x5118f8 0x549576 0x604173 0x5f5506 0x5f8c6c 0x5f9206
/usr/local/lib/python3.7/dist-packages/torch/utils/checkpoint.py:25: UserWarning: None of the inputs have requires_grad=True. Gradients will be None
  warnings.warn("None of the inputs have requires_grad=True. Gradients will be None")

Any idea where the issue is coming from?
